layers/+readers/djvu/img/djvu-logo.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
what's the license of this file

Reply:
Well, I can add the source of the image (I guess as a caption?), but don't know about any explicit permission.